src/components/organisms/AdminVenues/AdminVenues.scss

```diff
@@ -0,0 +1,99 @@
+@import "scss/constants.scss";
+
+$primary: #6f43ff;
```

Comment:
If I understand correctly this should be the primary color in the constants which is purplish instead of the blue current one. @mcflyok Is this statement correct?

Reply:
Yes! Sorry I've just seen this now !